client/my-sites/woocommerce/woocommerce-colophon.tsx

```diff
@@ -1,24 +1,29 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { useTranslate } from 'i18n-calypso';
-import ExternalLink from 'calypso/components/external-link';
 import WooCommerceLogo from './woocommerce-logo';
 
-function WooCommerceColophon() {
+export interface WooCommerceColophonProps {
+	wpcomDomain: string;
+}
+
+function WooCommerceColophon( props: WooCommerceColophonProps ) {
```

Comment:
```suggestion
function WooCommerceColophon( { wpcomDomain }: { wpcomDomain: string } ) {
```
I tend to just inline these when there's only a couple but it's up to you (sometimes exporting the props is helpful so makes sense to define them separately.